### ISSUE TITLE
Suggestions for Increasing W3C commitment to increased representation

### ISSUE THREAD

**Issue by tcweidner** [w3c/idcg#30]
Suggestion of creating a W3C membership requirement that member organizations meet certain diversity thresholds within their boards and senior leadership teams. 

Non-white male AC representation has been identified as a gap area. Fellowships and partnerships could be leveraged as one strategy to address it.

**Comment by tobie** [comment]
NASDAQ [just announced](https://www.businessinsider.com/nasdaq-companies-with-all-white-male-boards-face-delisting-2020-12) something similar.

**Comment by tcweidner** [comment]
Yes, very exciting. Nice to see an organization like Nasdaq taking the lead on something like this. Huge impact!
